app/assets/stylesheets/additional_submission_information.scss

```diff
@@ -0,0 +1,46 @@
+body.additional_submission_informations {
+  font-family: "Source Sans Pro", sans-serif;
```

Comment:
Why do we need to redefine the font family here? We already have it set globally somewhere else, no?